python/cugraph/tests/test_multigraph.py

```diff
@@ -0,0 +1,96 @@
+import cugraph
+import networkx as nx
+from cugraph.tests import utils
+import pytest
+import gc
+import numpy as np
+
+
+@pytest.mark.parametrize("graph_file", utils.DATASETS)
+def test_multigraph(graph_file):
+    gc.collect()
+    cuM = utils.read_csv_file(graph_file)
```

Comment:
We're working on some new test fixtures that will clean up much of the boilerplate (iterate over a set of input files, create a graph obj, create a Nx graph obj, etc.) that's used in many tests.  Would you mind adding a `FIXME` saying to remove most of this test setup and migrate to the new test fixtures when available?

Reply:
Added a FIXME